java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5042_java.html

```diff
@@ -1,71 +1,93 @@
-<p>Expanding archive files is security-sensitive. For example, expanding archive files has led in the past to the following vulnerabilities:</p>
-<ul>
-  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1263">CVE-2018-1263</a> </li>
-  <li> <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16131">CVE-2018-16131</a> </li>
-</ul>
-<p>Applications that expand archive files (zip, tar, jar, war, 7z, ...) should verify the path where the archive's files are expanded and not trust
-blindly the content of the archive. Archive's files should not be expanded outside of the root directory where the archive is supposed to be expanded.
-Also, applications should control the size of the expanded data to not be a victim of Zip Bomb attack. Failure to do so could allow an attacker to use
-a specially crafted archive that holds directory traversal paths (e.g. ../../attacker.sh) or the attacker could overload the file system, processors
-or memory of the operating system where the archive is expanded making the target OS completely unusable.</p>
-<p>This rule raises an issue when code handle archives. The goal is to guide security code reviews.</p>
+<p>Successful Zip Bomb attacks occur when an application expands untrusted archive files without controlling the size of the expanded data, which can
```

Comment:
S5042 should not be updated due to SONARJAVA-3532.

Reply:
reverted 👍🏻